### ISSUE TITLE
Java 8 Windows CI status failing

### ISSUE THREAD

**Issue by rajatbhatta** [googleapis/java-spanner#2205]
Java 8 Windows CI Status is "Failing".

Error:

```
[ERROR] /T:/src/github/java-spanner/google-cloud-spanner/src/main/java/com/google/cloud/spanner/nativeimage/SpannerFeature.java:[20,38] cannot access org.graalvm.nativeimage.hosted.Feature
  bad class file: C:\Users\kbuilder\.m2\repository\org\graalvm\sdk\graal-sdk\22.3.0\graal-sdk-22.3.0.jar(org/graalvm/nativeimage/hosted/Feature.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

Seems to be related to native images.

**Comment by mpeddada1** [comment]
@rajatbhatta Hm this looks familiar. It was previously addressed through https://github.com/googleapis/java-spanner/pull/1870/files so I'm curious as to why we're seeing this. Do you happen to have the PR that is running into this?

**Comment by rajatbhatta** [comment]
@mpeddada1: Thanks for looking into this. We've this latest nightly build that failed: https://source.cloud.google.com/results/invocations/47498ffc-9e1e-4c2f-916e-3453ff20b7ae/targets/cloud-devrel%2Fclient-libraries%2Fjava%2Fjava-spanner%2Fnightly%2Fjava8-win.

**Comment by mpeddada1** [comment]
Thank you! It looks like the `JAVA11_HOME` and `JAVA8_HOME` environment variables are not being set for this job. Troubleshooting continued at #2207.

**Comment by mpeddada1** [comment]
Closed via #2207 and #2210
